backends/fs/chroot/chroot-fs.cpp

```diff
@@ -42,7 +42,7 @@ bool ChRootFilesystemNode::exists() const {
 	return _realNode->exists();
 }
 
-Common::String ChRootFilesystemNode::getDisplayName() const {
+Common::U32String ChRootFilesystemNode::getDisplayName() const {
 	return getName();
```

Comment:
```suggestion
	return _realNode->getDisplayName();
```
Additionally, the implementation of `getName()` looks like it should probably return `_realNode->getName()`.